samples/ordered_puzzle/ordered_puzzle.html

```diff
@@ -30,11 +30,12 @@
         ></a-entity>
         <a-entity
           id="ordered-puzzle"
-          ordered-puzzle='jsonData: {"images": [{"imageSrc" : "./images/pic1.png", "width": 3, "height": 3, "xTarget": -7, "yTarget": 0, "x": 0, "y": 4},
-          {"imageSrc" : "./images/pic2.png", "width": 3, "height": 3, "xTarget": -3.5, "yTarget": 0, "x": -2, "y": -4},
-          {"imageSrc" : "./images/pic3.png", "width": 3, "height": 3, "xTarget": 0, "yTarget": 0, "x": -4, "y": 4},
-          {"imageSrc" : "./images/pic4.png", "width": 3, "height": 3, "xTarget": 3.5, "yTarget": 0, "x": 2, "y": -4},
-          {"imageSrc" : "./images/pic5.png", "width": 3, "height": 3, "xTarget": 7, "yTarget": 0, "x": 4, "y": 4}]}'
+          ordered-puzzle='jsonData: {"images": [{"imageSrc" : "./images/pic1.png", "width": 3, "height": 3, "xTarget": -7, "yTarget": 0},
```

Comment:
Small nit for future reference: can you leave the "xTarget": -7, "yTarget": 0 stuff for all the pics in here? Just so that when integrating with the main postgres mock data, we have a reference and can toggle `useTargets` to quickly validate that both types of puzzle work?
Thanks!

Reply:
For sure, updated PR :)